privacyidea/lib/tokens/yubicotoken.py

```diff
@@ -51,7 +51,7 @@
 
 YUBICO_LEN_ID = 12
 YUBICO_LEN_OTP = 44
-YUBICO_URL = "http://api.yubico.com/wsapi/2.0/verify"
```

Comment:
The constant YUBICO_URL is imported in a test. This is why the test fails.
Search the code for YUBICO_URL.
I would leave the constant with this name and then I would...